pkg/customeradministration/jobs/types.go

```diff
@@ -0,0 +1,145 @@
+}
+
+// OrganizationOrganizationCreateAsyncJobResult - An async organization creation job result
+type OrganizationOrganizationCreateAsyncJobResult struct {
+	// The date and time the job was created in UTC
+	CreatedUtc string `json:"createdUtc"`
+	// The error message in case of job failure
+	ErrorMessage string `json:"errorMessage,omitempty"`
+	// The date and time the job was completed in UTC
+	FinishedUtc string `json:"finishedUtc,omitempty"`
+	// The id of the job
+	ID string `json:"id"`
+	// The status of the job
+	Status OrganizationOrganizationCreateJobResultStatusEnum `json:"status"`
+}
+
+// OrganizationOrganizationCreateAsyncOrganizationResult - An async organization creation result organization
+type OrganizationOrganizationCreateAsyncOrganizationResult struct {
+	// Organization ID
+	ID string `json:"id,omitempty"`
+	// Organization name
+	Name string `json:"name,omitempty"`
+}
+
+// OrganizationOrganizationCreateAsyncResult - An async organization creation result
+type OrganizationOrganizationCreateAsyncResult struct {
+	// Organization Creation Async Job Customer
+	Customer OrganizationOrganizationCreateAsyncCustomerResult `json:"customer,omitempty"`
+	// Organization Creation Async Job Result
+	Job OrganizationOrganizationCreateAsyncJobResult `json:"job"`
+	// Organization Creation Async Job Organization
+	Organization OrganizationOrganizationCreateAsyncOrganizationResult `json:"organization,omitempty"`
+}
+
+// OrganizationOrganizationCreateAsyncResultCollection - Organization create async result collection
+type OrganizationOrganizationCreateAsyncResultCollection struct {
+	// Organization create async results
+	Items []OrganizationOrganizationCreateAsyncResult `json:"items"`
+	// an opaque cursor to supply with subsequent requests to get the next page of results, null if there are no more pages
+	NextCursor string `json:"nextCursor,omitempty"`
+}
+
+// OrganizationOrganizationCreateAsyncResultFilterInput - A filter for organization create job results
+type OrganizationOrganizationCreateAsyncResultFilterInput struct {
+	// Filter organization create job results by customer ID
+	CustomerId *OrganizationOrganizationCreateJobCustomerIdInput `json:"customerId,omitempty"`
+	// Filter oganization create job results by id
+	JobId *OrganizationOrganizationCreateJobIdInput `json:"jobId,omitempty"`
+	// Filter organization create job results by status
+	Status *OrganizationOrganizationCreateJobStatusInput `json:"status,omitempty"`
+}
+
+// OrganizationOrganizationCreateJobCustomerIdInput - The customer id search for jobs
+type OrganizationOrganizationCreateJobCustomerIdInput struct {
+	// A customer id to search for jobs in
+	Eq string `json:"eq"`
+}
+
+// OrganizationOrganizationCreateJobIdInput - The id of the job to retrieve
+type OrganizationOrganizationCreateJobIdInput struct {
+	// An organization create job id
+	Eq string `json:"eq,omitempty"`
+	// A set of job ids to search for
+	In []string `json:"in"`
```

Comment:
This unfortunately is not supported by Tutone - and you're right, I find that the query fails when there is `Eq` with `omitempty` and `In` without an `omitempty` - experienced this a few days ago. We certainly can manually change this to `omitempty` in the code, but that would lead to a lot of manual overrides, which would lead to confusion when this package is Tutone generated sometime later. 

While I try finding a solution to this, I think the only way for now would be to use `In` instead of `Eq` (add the string to be queried to a list, and supply it to `In`)